reactive-modules/ratpack-rxjava/src/main/java/com/baeldung/rxjava/RatpackErrorHandlingApp.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.rxjava;
+
+import ratpack.error.ServerErrorHandler;
+import ratpack.rx.RxRatpack;
+import ratpack.server.RatpackServer;
+import rx.Observable;
+
+public class RatpackErrorHandlingApp {
```

Comment:
Since this is an app that the user can run, can we place a class-level comment stating what endpoint they should hit to see the behavior in action? e.g.:

```java
/**
  * Try hitting http://localhost:8080/some-url to see the error handler in action
  */
```